src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,226 @@
+
+        boolean containsThree = Storage.people.size() == 3;
+        assertTrue(containsThree, "Error. The list should contain 3 users.");
+
+        assertNotEquals(Storage.people.get(0), Storage.people.get(1), "Test failed! "
+                + "There are at least 2 identical Users in the list.");
+        assertNotEquals(Storage.people.get(0), Storage.people.get(2), "Test failed! "
+                + "There are at least 2 identical Users in the list.");
+        assertNotEquals(Storage.people.get(1), Storage.people.get(2), "Test failed! "
+                + "There are at least 2 identical Users in the list.");
+    }
+
+    @Test
+    void register_loginLengthRequired_Ok() {
+        user1.setLogin("abcdef");
+        User actual = registrationService.register(user1);
+        assertEquals(user1.getLogin(), actual.getLogin(), "Test failed! "
+                + "The user with login that is >= 6 characters should be registered.");
+
+        user2.setLogin("abcdefghijk");
+        actual = registrationService.register(user2);
+        assertEquals(user2.getLogin(), actual.getLogin(), "Test failed! "
+                + "The user with login that is >= 6 characters should be registered.");
+
+        user3.setLogin("abcdefghijkabcdefghijkabcdefghijk");
+        actual = registrationService.register(user3);
+        assertEquals(user3.getLogin(), actual.getLogin(), "Test failed! "
+                + "The user with login that is >= 6 characters should be registered.");
+
+        boolean containsThree = Storage.people.size() == 3;
+        assertTrue(containsThree, "Error. The list should contain 3 users.");
+    }
+
+    @Test
+    void register_passwordLengthRequired_Ok() {
+        user1.setPassword("abcdef");
+        User actual = registrationService.register(user1);
+        assertEquals(user1.getPassword(), actual.getPassword(), "Test failed! "
+                + "The user with password that is >= 6 characters should be registered.");
+
+        user2.setPassword("abcdefghijk");
+        actual = registrationService.register(user2);
+        assertEquals(user2.getPassword(), actual.getPassword(), "Test failed! "
+                + "The user with password that is >= 6 characters should be registered.");
+
+        user3.setPassword("abcdefghijkabcdefghijkabcdefghijk");
+        actual = registrationService.register(user3);
+        assertEquals(user3.getPassword(), actual.getPassword(), "Test failed! "
+                + "The user with password that is >= 6 characters should be registered.");
+
+        boolean containsThree = Storage.people.size() == 3;
+        assertTrue(containsThree, "Error. The list should contain 3 users.");
+    }
+
+    @Test
+    void register_atLeast18Age_Ok() {
+        user1.setAge(18);
+        User actual = registrationService.register(user1);
+        assertEquals(user1.getAge(), actual.getAge(), "Test failed! "
+                + "The user which is 18 years or more should be registered.");
```

Comment:
minor: you might want to pick message that is as short and as descriptive as possible. 
In case like this it might be more clear if you go with "Can't register user below 18"